client/js/domhelpers.js

```diff
@@ -72,11 +72,31 @@ export function formField(field, dom, id) {
 
   if(field.type == 'number') {
     const input = document.createElement('input');
+    const spanafter = document.createElement('span');
     input.type = 'number';
-    input.value = field.value !== undefined ? field.value : 1;
-    input.min = field.min !== undefined ? field.min : 1;
-    input.max = field.max !== undefined ? field.max : 10;
+    input.step = "any";
+    input.value = field.value !== undefined ? field.value : 0;
+    if(input.value > field.max){
+      input.value = field.max;
+    }
+    if(input.value < field.min){
+      input.value = field.min;
+    }
+    if(field.min && field.max){
+      label.innerHTML += " <span class='numberInputRange'>("+ field.min +" - "+ field.max +")</span>";
```

Comment:
Still putting user data into `innerHTML`.

Reply:
Since those could only be numbers I didn't see a security issue. But I did change it again. Should be fine now.